### ISSUE TITLE
batchnorm in resnet32

### ISSUE THREAD

**Issue by aliarastehfard** [mc2-project/delphi#49]
I noticed that batchnorm is currently missing from Delphi's neural network module. I'm curious about running inference for resnet32. Can you provide some guidance on how it works?

**Comment by aliarastehfard** [comment]
I'm wondering if a Batchnorm implementation in Rust is needed to load the pretrained model correctly. Do we need to implement Batchnorm in Rust under the rust/neural-network module?

**Comment by ryanleh** [comment]
During inference, [batch norm reduces to a linear transform](https://en.wikipedia.org/wiki/Batch_normalization#Inference)–since it's performed after convolution (also a linear transformation), you can just [merge them both into a single convolution operation](https://towardsdatascience.com/speed-up-inference-with-batch-normalization-folding-8a45a83a89d8).
